spec/parse_date/int_from_string_spec.rb

```diff
@@ -698,7 +698,7 @@
       end
     end
     it 'true for 0000' do
```

Comment:
title's now out of sync with test, but wouldn't hold up merge since you said this is a prod issue